File: sdk/java/src/main/java/ai/aerol/microvm/internal/api/v1/Paths.java
package ai.aerol.microvm.internal.api.v1;

/**
 * v1 path-prefix constant for the Java SDK.
 *
 * <p>Mirrors the prefix exported by {@code pkg/api/v1/dto.go::PathPrefix} on
 * the server. When a new wire version lands, a sibling
 * {@code ai.aerol.microvm.internal.api.vN.Paths} will export its own
 * {@code PATH_PREFIX}, and {@code MicroVMConfig.apiVersion} selects which one
 * to use.
 */
public final class Paths {
    public static final String PATH_PREFIX = "/v1";

    private Paths() {}
}

File: sdk/python/microvm/_internal/api/v1/paths.py
# v1 path-prefix constant for the Python SDK. Mirrors the prefix exported by
# pkg/api/v1/dto.go::PathPrefix on the server. When a new wire version
# lands, a sibling microvm/_internal/api/vN/paths.py will export its own
# PATH_PREFIX, and the MicroVM(api_version=...) option selects which one to
# use.
PATH_PREFIX = "/v1"
